action.yml

```diff
@@ -5,4 +5,4 @@ branding:
   color: orange
 runs:
   using: "node20"
-  main: "dist/index.js"
+  main: "index.js"
```

Comment:
are you sure this is working? Others won't be able to use the action, the whole code must be bundled for it to work